modules/ROOT/pages/setting-environment-variables.adoc

```diff
@@ -0,0 +1,26 @@
+= Setting Environment Variables
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, environment variables, system variables, system properties
+
+If you're deploying multiple applications as a shared resource, you shouldn't set any properties in the project properties files. This might cause conflicts between the various apps that share a domain. Instead, you can set environment variables over the scope of the deployed app, its domain, and other apps under that domain.
```

Comment:
I'd suggest to avoid using conditionals as should, might. Do you think it would be better if we replace with the following: 
"you shouldn't set" > "you must not set"
"This might cause" > "This can cause" 
"Instead, you can set" > "Instead, set"

Reply:
I think shouldn't is OK in this context, but I agree with you on the other two comments.

Reply:
I ended up using "it is recommended that..."